server/testutil/webtester/webtester.go

```diff
@@ -77,21 +77,16 @@ func (wt *WebTester) screenshot(tag string) error {
 	// TEST_UNDECLARED_OUTPUTS_DIR is usually defined by Bazel. If this test is
 	// run outside of Bazel for whatever reason, this will just be an empty
 	// string, which is interpreted by CreateTemp as "use the OS-default temp dir"
-	testScreenshotDir := os.Getenv("TEST_UNDECLARED_OUTPUTS_DIR")
-	screenshotFile, err := os.CreateTemp(testScreenshotDir, fmt.Sprintf("%s.%s.screenshot.png", wt.t.Name(), tag))
-	if err != nil {
-		return err
-	}
-	defer screenshotFile.Close()
+	screenshotFile := testfs.MakeTempFile(wt.t, os.Getenv("TEST_UNDECLARED_OUTPUTS_DIR"), fmt.Sprintf("%s.%s.screenshot-*.png", wt.t.Name(), tag))
```

Comment:
I think we still want to use `os.CreateTemp` here -- IIUC `MakeTempFile` will delete the screenshot automatically on test cleanup

Reply:
(sorry, caught this after approving)

Reply:
lol np, i'll fix it.